x-pack/plugin/core/src/main/resources/searchable-snapshots-template.json

```diff
@@ -0,0 +1,31 @@
+{
+  "index_patterns": [
+    ".snapshots-${xpack.searchable_snapshots.template.version}"
+  ],
+  "priority": 100,
+  "version": ${xpack.searchable_snapshots.template.version},
+  "data_stream": {
+  },
+  "composed_of": [
+  ],
+  "template": {
+    "aliases": {
+      ".snapshots": {}
+    },
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 0,
+      "auto_expand_replicas": "0-1"
+    },
+    "mappings": {
+      "_doc": {
+        "_meta": {
+          "searchable-snapshot-version": "${xpack.searchable_snapshots.version}"
+        },
+        "dynamic": false,
+        "properties": {
+        }
+      }
+    }
+  }
```

Comment:
Can you add:

```json
  "_meta": {
    "managed": true,
    "description": "a good description of this template"
  }
```

To this? The `managed: true` ensures that the UI has a little "managed" badge by it

Reply:
Good to know, thanks!